src/chordsymbol.ts

```diff
@@ -13,10 +13,11 @@ import { Font, FontInfo, FontStyle, FontWeight } from './font';
 import { Glyph } from './glyph';
 import { Modifier } from './modifier';
 import { ModifierContextState } from './modifiercontext';
+import { Note } from './note';
 import { StemmableNote } from './stemmablenote';
 import { Tables } from './tables';
 import { TextFormatter } from './textformatter';
-import { Category } from './typeguard';
+import { Category, isStemmableNote } from './typeguard';
 import { log } from './util';
 
```

Comment:
I think that you included these changes by mistake. Also the other ones in this file.